sim/src/make/scenario.rs

```diff
@@ -145,7 +145,8 @@ impl Scenario {
             timer.next();
 
             if let Err(err) = p.check_schedule() {
-                panic!("{}", err);
+                error!("skipping invalid schedule: {}", err);
```

Comment:
Don't miss this bit! I haven't looked into why this is happening yet, but skipping these invalid one still results in lots of trips being made.

I suppose we need some kind of disqualifying criteria to prevent these kinds of schedules from being created upstream, but for now I just switched it to an error log.

Reply:
I'm hesitant to remove this check, because there are other places that create scenarios. And if a weird schedule gets through, I don't remember how the sim spawning layer handles it -- I think there'll be some assertion failure when the trip transition happens, which is a pretty deferred way to discover a bug.

Could we use `scenario.remove_weird_schedules()` instead? And maybe update the `println`s there to log

Reply:
Didn't know about remove_weird_schedules - thanks. Done!